File: python solutions/1.py
class Solution:
  def twoSum(self, nums: list, target: int) -> list:
    
    for i in range(len(nums)): 
      for j in range(i+1, len(nums)): 
        if nums[i] + nums[j] == target: 
          return [i, j]  

File: cpp solutions/1.cpp
#include <vector> 
class Solution {
public:
    std::vector<int> twoSum(std::vector<int>& nums, int target) {
        std::vector<int> tmp; 

        for(int i = 0; i < nums.size(); i++){ 
          for(int j = i+1; j < nums.size(); j++){ 
            if (nums[i] + nums[j] == target) { 
              tmp.push_back(i); 
              tmp.push_back(j); 
              return tmp; 
            }
          }
        }
      return tmp; 
    }
};

int main(){ 
  return 0; 
}